src/app/feature/map/map.component.html

```diff
@@ -14,7 +17,5 @@
   <app-whereabouts class="whereabouts"></app-whereabouts>
 </div>
 <div class="coordinates" *ngIf="isDevMode && x && y">
```

Comment:
Please, try to move such logic isDevMode && x && y in service or component getter